composer.json

```diff
@@ -13,9 +13,8 @@
         "php": ">=5.4.0",
         "ext-openssl": "*",
         "ext-mcrypt": "*",
-        "ext-mbstring": "*",
```

Comment:
No. The main composer.json doesn't inherit the others. It just totally replaces them. They aren't even looked at by composer.